presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatistics.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.statistics;
+
+import java.util.Optional;
+
+public class ColumnStatistics
+{
+    public static final ColumnStatistics EMPTY_STATISTICS = ColumnStatistics.builder().build();
+
+    private final Optional<StatisticsValue<Long>> dataSize;
+    private final Optional<StatisticsValue<Long>> rowsCount;
+    private final Optional<StatisticsValue<Object>> min;
+    private final Optional<StatisticsValue<Object>> max;
```

Comment:
I don't see value in grouping them but there could be some value in providing partial stats.
E.g. scenario: connector is assuming DB is used mainly through Presto so stats are refreshed on inserts/updates through Presto but from time to time they are refreshed from remote system (in case there are outside Presto DMLs).
So refresh time = last refresh / DML + TTL
Assuming `min` value was established recently by insert through Presto but no one updated `max` value using Presto for a while. Then stats for `min` value can be provided but unless connector refreshes stats from remote system `max` can't be provided.

It may not be a real case but shows there's some explanation for such an approach. And given the fact there's no benefit of merging them together I'd keep them separate.